{
  "_annotation": {
    "cna": "psf",
    "cve_id": "CVE-2025-8291",
    "description": "The 'zipfile' module would not check the validity of the ZIP64 End of\nCentral Directory (EOCD) Locator record offset value would not be used to\nlocate the ZIP64 EOCD record, instead the ZIP64 EOCD record would be\nassumed to be the previous record in the ZIP archive. This could be abused\nto create ZIP archives that are handled differently by the 'zipfile' module\ncompared to other ZIP implementations.\n\n\nRemediation maintains this behavior, but checks that the offset specified\nin the ZIP64 EOCD Locator record matches the expected value.",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2025/CVE-2025-8291.json",
    "modified": "2025-10-07T18:22:46.912Z",
    "published": "2025-10-07T18:10:05.908Z",
    "reason": "Added CPE configurations because not yet analyzed by NVD.",
    "references": [
      "https://github.com/python/cpython/commit/162997bb70e067668c039700141770687bc8f267",
      "https://github.com/python/cpython/commit/333d4a6f4967d3ace91492a39ededbcf3faa76a6",
      "https://github.com/python/cpython/issues/139700",
      "https://github.com/python/cpython/pull/139702",
      "https://mail.python.org/archives/list/security-announce@python.org/thread/QECOPWMTH4VPPJAXAH2BGTA4XADOP62G/"
    ]
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:python:python:*:*:*:*:*:*:*:*",
                "matchCriteriaId": "38DCA733-D883-5DEB-8330-5E68BAF465C7",
                "versionEndIncluding": "3.14.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}